src/BenchmarkDotNet/BenchmarkDotNet.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\..\build\common.props" />
   <PropertyGroup>
     <AssemblyTitle>BenchmarkDotNet</AssemblyTitle>
-    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net6.0;net8.0</TargetFrameworks>
```

Comment:
We just merged the update to .NET 8 SDK in #2462, could you please sync your branch?

Reply:
Merged but this doesn't change this file?

Reply:
> Merged but this doesn't change this file?

Correct.